modules/users/server/controllers/users/users.authentication.server.controller.js

```diff
@@ -83,18 +83,17 @@ exports.signout = function (req, res) {
 /**
  * OAuth provider call
  */
-exports.oauthCall = function (strategy, scope) {
-  return function (req, res, next) {
+exports.oauthCall = function (req, res, next) {
+    var strategy = req.params.strategy;
     // Authenticate
-    passport.authenticate(strategy, scope)(req, res, next);
+    passport.authenticate(strategy)(req, res, next);
```

Comment:
Hey @securityvoid, I've just read the change again. I see what you did. I somehow assumed that I was seeing part of the config file but you actually set up the scope in the `passport.use()` call instead whilst creating the strategy. When I woke the next day I didn't re-read.

Nice! I've not seen scope set up anywhere else I dug though the source just to make sure it would be safe to rely on the use of [strategy](https://github.com/jaredhanson/passport-oauth2/blob/master/lib/strategy.js#L95) on the `method` for other strategies.

It should work in any strategy as long as they inherit from his OAuth2Strategy (as he does)[https://github.com/jaredhanson/passport-facebook/blob/master/lib/strategy.js#L63] when he created most of the common strategies or if they create a custom one with scope (which should be most strategies).